src/HtmlProcessor/AbstractHtmlProcessor.php

```diff
@@ -228,7 +265,8 @@ private function createRawDomDocument(string $html)
     private function prepareHtmlForDomConversion(string $html): string
     {
         $htmlWithSelfClosingSlashes = $this->ensurePhpUnrecognizedSelfClosingTagsAreXml($html);
-        $htmlWithDocumentType = $this->ensureDocumentType($htmlWithSelfClosingSlashes);
+        $htmlWithRootElms = $this->addMissingRootElements($htmlWithSelfClosingSlashes);
```

Comment:
Nitpicky, but we would prefer `Elements` to `Elms` - if it's good enough for a method name it's good enough for a variable :)